database/pgsql/vulnerability.go

```diff
@@ -220,17 +219,6 @@ func (tx *pgSession) insertVulnerabilities(vulnerabilities []database.Vulnerabil
 	return vulnIDs, nil
 }
 
-// castMetadata marshals the given database.MetadataMap and unmarshals it again to make sure that
-// everything has the interface{} type.
-// It is required when comparing crafted MetadataMap against MetadataMap that we get from the
-// database.
-func castMetadata(m database.MetadataMap) database.MetadataMap {
-	c := make(database.MetadataMap)
-	j, _ := json.Marshal(m)
-	json.Unmarshal(j, &c)
-	return c
-}
```

Comment:
Why don't we need this anymore?

Reply:
Wait, No, we don't need it. We will not compare crafted MetadataMap against MetadataMap from database query result any more because Updater will only compare MetadataMaps from database.